frame/transaction-payment/src/lib.rs

```diff
@@ -103,6 +217,38 @@ decl_module! {
 				*fm = T::FeeMultiplierUpdate::convert(*fm);
 			});
 		}
+
+		fn on_runtime_upgrade() -> Weight {
```

Comment:
as the computation of fee changed, is the current Multiplier a good value for new computation mecanism ?

Reply:
So the migration should do the following: The old multiplier is read. if it is negative it will be set to zero. Else, it will be set to its old value in FixedU128 (we could improve this by setting it to `old + 1` actually. But it doesn't matter much). 

We know that in our chains including Kusama the current value is `-1` so it will be read from storage, then it will set to 0, then the first `convert()` will do a `.max(min_multiplier)` on this which will set it to a small number defined in the companion PR.

Reply:
ok so we are good starting with min_multiplier for our chains